### ISSUE TITLE
Add new skins

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#2193]
- #2152 
- ...

**Comment by Zatline** [comment]
How am I not assigned to this?

**Comment by AssassinTee** [comment]
![https://i.imgur.com/CMETPz8.png](https://i.imgur.com/CMETPz8.png)
tripple heart
![https://i.imgur.com/ljJuRfX.png](https://i.imgur.com/ljJuRfX.png)
single heart
![https://i.imgur.com/gt5QIpg.png](https://i.imgur.com/gt5QIpg.png)
some screens (sorry for the bad cropping)

What do you think?

**Comment by oy** [comment]
Hearts are a good idea 👍

**Comment by Dune-jr** [comment]
@Sonix-'s skins: https://www.teeworlds.com/forum/viewtopic.php?id=11530
@Zatline's skins: https://www.teeworlds.com/forum/viewtopic.php?id=7820

> both are pretty good quality. I think some of them might be slightly too much/too big for vanilla / disturbing for the aim

**Comment by Dune-jr** [comment]
```
0.240KB	data/skins
0.340KB	data/maps
0.696KB	data/ui
0.708KB	data/fonts
0.712KB	data/editor
0.892KB	data/languages
1.064KB	data/countryflags
3.332KB	data/mapres
4.212KB	data/audio
12.476KB data
```

Skins are quite light, there is room for a bunch :) @Zatline

**Comment by oy** [comment]
Fixed by #2315
